scripts/install_avalanchego_release.sh

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env bash
```

Comment:
We once had this copied over from `subnet-evm`, but got rid of it for some reason. Are we not able to use the file directly from our submodule while setting some env vars? Would it possibly be worthwhile to make a PR to `subnet-evm` to make it usable if not? Just spitballing :)

Reply:
We use it like this in teleporter. I think scripts like this are reasonable to have copy pastad in individual repos IMO  and we do it already with other scripts. Happy to be overridden though.

Reply:
Oh, lol forgot which repo I was in. Can we use the one from the teleporter submodule? I don't mind copying code if its the best/easiest option, just wondering if we can piggyback somewhere here

Reply:
each of them uses their own `versions.sh` we certainly can but then we end up with the same problem of coupling repos tighter than they need to be

Reply:
Yeah, that's fair, you've convinced me 😅